packages/nextjs/app/admin/_components/SubmissionCard.tsx

```diff
@@ -87,89 +87,126 @@ export const SubmissionCard = ({ submission }: { submission: Submission }) => {
   const currentVote = submission.votes.find(vote => vote.builder === connectedAddress);
   const score = currentVote ? currentVote.score : 0;
 
+  const telegramUser = submission.telegram?.replace("@", "");
+
   return (
-    <div key={submission.id} className="indicator">
-      <div className="indicator-item badge badge-secondary flex flex-col p-8">
-        <div className="text-2xl font-bold">{scoreAvg}</div>
-        <div>{submission.votes.length} votes</div>
-      </div>
-      <div className="card bg-primary text-primary-content">
-        <div className="card-body">
-          <div className="flex mb-4 items-center">
-            <div className="rating flex items-center">
+    <div key={submission.id} className="card bg-secondary text-secondary-content border border-primary rounded-none">
+      <div className="card-body p-5 xl:p-6">
+        <h2 className="card-title mb-4 text-2xl xl:text-3xl">{submission.title}</h2>
+        <div className="flex flex-wrap justify-between items-center gap-4">
+          <div className="mt-1 flex shrink-0 gap-3">
+            {submission.linkToRepository && (
+              <a href={submission.linkToRepository} className="inline-block" target="_blank">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  stroke="currentColor"
+                  strokeWidth="2"
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                >
+                  <path d="M15 22v-4a4.8 4.8 0 0 0-1-3.5c3 0 6-2 6-5.5.08-1.25-.27-2.48-1-3.5.28-1.15.28-2.35 0-3.5 0 0-1 0-3 1.5-2.64-.5-5.36-.5-8 0C6 2 5 2 5 2c-.3 1.15-.3 2.35 0 3.5A5.403 5.403 0 0 0 4 9c0 3.5 3 5.5 6 5.5-.39.49-.68 1.05-.85 1.65-.17.6-.22 1.23-.15 1.85v4" />
+                  <path d="M9 18c-4.51 2-5-2-7-2" />
+                </svg>
+              </a>
+            )}
```

Comment:
Maybe we can add this (and the video one) as an SVG file as you did with the telegram one? Or even a tsx file if we need to have props on it.

Reply:
Got this moved to a file